.gitignore

```diff
@@ -10,3 +10,6 @@ syntax: glob
 *.egg-info
 dist
 build
+
+# Pycharm
+.idea
```

Comment:
These are IDE/editor-specific files, which differ from user to user, as our personal choices are diverse. The right place for these – and many others – is the global ignore file `~/.config/git/ignore`, following the XDG Base Directory Specification. `man gitignore` has detailed information.